xyz-hub-service/src/main/java/com/here/xyz/hub/task/ModifySpaceOp.java

```diff
@@ -37,14 +37,14 @@
 public class ModifySpaceOp extends ModifyOp<Space, SpaceEntry> {
 
   public ModifySpaceOp(List<Map<String, Object>> inputStates, IfNotExists ifNotExists, IfExists ifExists, boolean isTransactional) {
-    super((inputStates == null) ? Collections.emptyList() : inputStates.stream().map(SpaceEntry::new).collect(Collectors.toList()),
-        ifNotExists, ifExists, isTransactional);
+    super((inputStates == null) ? Collections.emptyList() : inputStates.stream().map(input -> new SpaceEntry(input, ifNotExists, ifExists))
+            .collect(Collectors.toList()), ifNotExists, ifExists, isTransactional);
```

Comment:
Removing here "ifNotExists, ifExists", will enable the removal on ModifyOp.

Reply:
Done.